### PR TITLE
chore: add take action for contributors to self-assign

### LINKED ISSUE #6: Add the take action
I'm hoping we could add the take action, so that contributors can assign themselves to an issue if they comment the phrase ".take"

https://github.com/bdougie/take-action

### PR DESCRIPTION
**Pull request by blackgirlbytes**
This workflow should enable contributors to self-assign issues.

This pull request closes #6

### CODE REVIEW

File: .github/workflows/take.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# .github/workflows/take.yml
```
Comment (JoshuaKGoldberg) [inline .github/workflows/take.yml:1]: Nit: any reason for this?

```suggestion
```

**Review by JoshuaKGoldberg — APPROVED** [review]
🚀 great!

File: .github/workflows/take.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    name: Take an issue
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+    - name: take the issue
+      uses: bdougie/take-action@main
+      with:
+        message: Thanks for taking this issue! Let us know if you have any questions!
+        trigger: .take
```
Comment (JoshuaKGoldberg) [inline .github/workflows/take.yml:17]: You know, on second glance, this should probably be documented in the repo's development docs. @blackgirlbytes do you want to do that in this PR or a followup?

Comment (blackgirlbytes) [reply .github/workflows/take.yml:16]: in a follow up if that's okay!

**Comment by blackgirlbytes** [comment]
okay, I'm lost. Why is prettier failing for this @JoshuaKGoldberg

**Pull request merged**